### PR TITLE
Mark TODOs in content

### PR DESCRIPTION
**Pull request by ksuess**
````
```{admonition} TODO
:class: warning

Add content for this section
```
````

image

### CODE REVIEW

**Comment by stevepiercy** [comment]
Good catch!
<details><summary>Browser metadata</summary>





```
Path:      /
Browser:   Chrome 97.0.4692.71 on Mac OS 10.15.7
Viewport:  1309 x 939 @1x
Language:  en-US
Cookies:   Enabled
```

[Open in BrowserStack](https://www.browserstack.com/user/try-live?url=https%3A%2F%2Fdeploy-preview-1166--6-dev-docs-plone-org.netlify.app%2F&os=OS%20X&os_version=Catalina&browser=Chrome&browser_version=97.0&resolution=1309x939&speed=1&start=true&ref=netlify-source)



</details>

[Open Deploy Preview](https://deploy-preview-1166--6-dev-docs-plone-org.netlify.app/) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=61efb1556cebba0007a27040&commentID=61efc7e0ea34bfd35f8d27dd&resolution=resolved)

File: docs/classic-ui/index.md [hunk 1]
```diff
@@ -10,7 +10,9 @@ html_meta:
 
 # Classic UI
 
-```{todo}
+```{admonition} TODO
+:class: warning
+
```
Comment (stevepiercy) [inline docs/classic-ui/index.md]: I don't think any of these changes to MyST markup are necessary. The current MyST markup generates the following HTML:

```html
<div class="admonition-todo admonition" id="id1">
<p class="admonition-title">Todo</p>
<p>Perhaps some of this introduction should be an include, so we don't have to repeat ourselves?</p>
</div>
```

Here is the default.

Screen Shot 2022-01-30 at 5 48 44 PM

We could change the class `admonition-todo` to something that implies todo-ness.

Screen Shot 2022-01-30 at 6 19 38 PM

## `docs/_static/custom.css`
```css

/* Add Font Awesome 5 icon for todo */
:root {
  --pst-icon-clipboard-list: '\f46d';
  --pst-icon-admonition-todo: var(--pst-icon-clipboard-list);
}

.admonition > .admonition-todo:before, div.admonition-todo > .admonition-title:before {
  content: var(--pst-icon-admonition-todo);
}
```

Please let me know. Thank you!

**Comment by stevepiercy** [comment]
See https://github.com/plone/documentation/issues/1165

**Comment by ksuess** [comment]
Catchy color, hoping the todos are eliminated before relaunch.

**Comment by ksuess** [comment]
preview on page with todo: https://deploy-preview-1166--6-dev-docs-plone-org.netlify.app/classic-ui/index.html

**Review by stevepiercy — APPROVED** [review]
I like lavender. :grin:

**Comment by stevepiercy** [comment]
@jenkins-plone-org please run jobs

**Comment by ksuess** [comment]
> I like lavender. 😁

Then it's the wrong color, should be ugly.

**Comment by stevepiercy** [comment]
> > I like lavender. 😁
> 
> Then it's the wrong color, should be ugly.

Oh, no! I meant it is hideous! 🤮

**Comment by stevepiercy** [comment]
I have no clue what Jenkins is supposed to do. Please go ahead and merge when you are ready to do so.

**Pull request merged**